sdp/processors/modify_manifest/speech_recognition.py

```diff
@@ -0,0 +1,126 @@
+import json
+from tqdm import tqdm
+from pathlib import Path
+from sdp.processors.base_processor import BaseProcessor
+from sdp.utils.common import load_manifest
+
```

Comment:
Users will probably get confused about why the NeMo ASR processors are not in this file as well.

Reply:
moved to huggingface folder